lib/resources/example_json/r4_examples_observation.rb

```diff
@@ -0,0 +1,2458 @@
+module Cerner
+    module Resources
+  
+        R4_OBSERVATION_BUNDLE ||=
+        {
```

Comment:
this example is way too large to display.

Reply:
trimmed it down: https://github.com/cerner/fhir.cerner.com/pull/345/commits/85bb1a56bd4635bc292f0fbe1300685b9970981e#diff-cd059d38e44f892deed0ab35bef7831cL501